app/views/directory_objects/placard.html.erb

```diff
@@ -2,181 +2,93 @@
-			if(numLines < 5){
-				document.getElementById("extraField").innerHTML= infoFormat;
-				document.getElementById("errorField").style.visibility= "hidden";
-			}
-			else{
-				document.getElementById("errorField").style.visibility= "visible";
-				document.getElementById("extraField").innerHTML= "";
-			}
-			}
-			
-	</script>
-
-
-  		<!-- ========= Start Placard ============-->
-
-    <% @results.each do |person| %>
-		<!-- This is the placard background -->
-		<div class="backgroundImg">
-      		 <%= image_tag("placard/back0.jpg", id: "bgImage", alt: " ", class: "bgstyle") %>
-
-			<!--WATCH FOR <DIV> NESTING-->
-
-			<!-- This is the name field -->
-			<div id="nameField" class="name"> <%= person[:name] %>
-
-				<div id="titleField" class="title"><%= person[:title] %> </div>
-
-				<div id = "extraInfoLine" class = "leftLine">
-					<!-- This is the email address-->
-					<div id="emailField" class="email"><%= person[:email] %></div>
-
-					<!--Extra info-->
-					<div id="extraField" class="extrainfo"></div>
-				</div>
+<body>
 
-			</div>
+	<!-- ========= Start Placard ============-->
 
-			<!-- This is the QR image-->
-			<div class="qr"> <%= image_tag(person[:qrLink], alt: "qrCode", class: "qrstyle", id: "qrImage") %></div>
+	<% @results.each do |person| %>
+	<!-- This is the placard background -->
+	<div class="backgroundImg">
+		<%= image_tag("placard/back0.jpg", id: "bgImage", alt: " ", class: "bgstyle") %>
 
-			<!-- This is the school logo -->
-			<div class="logo"><%= image_tag("placard/davislogo.png", alt: "logoCode", class: "logostyle", id: "logoImage") %></div>
+		<!--WATCH FOR <DIV> NESTING-->
 
-			<!-- This is the department name-->
-			<div id="depNameField" class="depname"><%= person[:department] %></div>
+		<!-- This is the name field -->
+		<div id="nameField" class="name"> <%= person[:name] %>
 
-		</div>
-	    <% end %>
-		<!-- ========= End Placard ============-->
+			<div id="titleField" class="title"><%= person[:title] %> </div>
 
-		<!-- This is where the user can change the background-->
-		<h3> Optional Customization</h3>
+			<div id = "extraInfoLine" class = "leftLine">
+				<!-- This is the email address-->
+				<div id="emailField" class="email"><%= person[:email] %></div>
+
+				<!--Extra info-->
+				<!-- <h3>Office Hours</h3> -->
+				<div id="extraField" class="extrainfo">Office Hours
+					<span><%= simple_format(person[:office_hours]).html_safe %></span>
+				</div>
+			</div>
 
-		<div class= "buttonDiv">
-		<p id="backgroundTitle" class= "hidep">
-		<button onclick="changeBgImage(0)" id = "0">Previous Image</button>
-		<button onclick="changeBgImage(1)" id = "1">Next Image</button>
-		Change Background</p>
 		</div>
 
-		<!-- This is where the user can change the title-->
-		<input type="text" id="newTitle" size="35" placeholder="Enter your title"> 
-		<br>
+		<!-- This is the QR image-->
+		<div class="qr"> <%= image_tag(person[:qrLink], alt: "qrCode", class: "qrstyle", id: "qrImage") %></div>
```

Comment:
Could person[:qrLink] be taken straight from a method in the DirectoryObject model class, e.g. DirectoryObject.qr_code?